students/JamesTraub/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+# Lesson 3 - Series 1
+
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print("Create and display a list of fruit.")
+print(fruit)
+print()
+
+inp_fruit = input('Please enter a name of a fruit to add to the end of list. ')
+fruit.append(inp_fruit)
+print(fruit)
+print()
+
+fruit_number = int(input('Please enter the number that correspends with one fruit in the list. '))
+for i, nfruit in enumerate(fruit):
+    if fruit_number == (i + 1):
+        print(fruit_number, nfruit)
+print()
+
+add_fruit2 = []
+inp_fruit2 = input('Please enter a fruit to add to the beginning of the list. ')
+fruit = [inp_fruit2,] + fruit
```

Comment:
you only need the comma in the tuple -> `(inp_fruit2,)` but in list its just `[inp_fruti2]`